client/state/preferences/actions.js

```diff
@@ -28,26 +28,35 @@ export function fetchPreferences() {
 }
```

Comment:
>They are extremely rarely updated, which leads to comments that lie

Surfaced by ESLint, evidenced by my original comment here.

>Information they provide is often useless because it just repeats the parameters and we are not using them to generate any documentation

It's not just repeating parameters, it's:

- Detailing the intended behavior and arguments in human-readable sentences
- Providing expected forms (types) and nullability of parameters and return value

To me, this seems like arguing the merits of documenting code at all. The point is being considerate of future developers who don't have the same intimate familiarity with the inner workings that you have. Obviously there are varying definitions of what constitutes acceptable level of documentation (vs. excessive), which I think is fine to discuss... as long as it _is_ being discussed. "Useless", however, is very much a stretch 😄 

>They encourage bad code because they obscure the fact that code underneath is bad

I'm not sure what this means. That presence of documentation might incorrectly lead someone to assume a certain level of code quality?